Server/src/services/fileUploadService.ts

```diff
@@ -97,10 +100,10 @@ export class fileUploadService {
 
     //checks and validates if ref authors are strings and handles error--
     referenceAuthors = jsonObj.reference.authors;
-
     this.arrTypeValidationCheck(referenceAuthors, 'string');
+    let allAuthors: any[];

```

Comment:
Is this mappable to an Authors Interface or type? Like IAuthors

Reply:
This array will specifically hold Authors table/entry objects. I'll see if I can specify that as the array type, uncertain if possible.

Reply:
Authors[] works as a type; change made